domain-management/pom.xml

```diff
@@ -71,6 +71,10 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.aesh</groupId>
```

Comment:
I will let @bstansberry also comment on this dependency.

IMO I am not overly concerned as the add-user utility is destined to be deprecated and either re-written or replaced hopefully soon.